src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -116,6 +116,32 @@ def _webhook_event_data(
     return event_context
 
 
+@instrumented_task(
+    name="sentry.sentry_apps.tasks.sentry_apps.send_alert_webhook_v2", **TASK_OPTIONS
+)
+@retry_decorator
+def send_alert_webhook_v2(
+    rule_label: str,
+    sentry_app_id: int,
+    instance_id: str,
+    group_id: int,
+    occurrence_id: str,
+    additional_payload_key: str | None = None,
+    additional_payload: Mapping[str, Any] | None = None,
+    **kwargs: Any,
+):
+    send_alert_webhook(
+        rule=rule_label,
+        sentry_app_id=sentry_app_id,
+        instance_id=instance_id,
+        group_id=group_id,
+        occurrence_id=occurrence_id,
+        additional_payload_key=additional_payload_key,
+        additional_payload=additional_payload,
+        **kwargs,
+    )
+
+
```

Comment:
This block alongside adding the new task to the import list are the only "new" changes in this PR for the sake of the cutover.

Reply:
should this new task be used in the PR? can you remind me how the cutover happens

Reply:
Cutover has 3 steps
1. Create new task stub and have it call the old task (this is the forward compatible task) - 1st PR
2. Move the logic over from the old task to the new task, all callsites should now use the new task -2nd PR
3. Delete old tasks